### PR TITLE
Add legacy emboss 5.0 suite

### PR DESCRIPTION
**Pull request by mvdbeek**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/emboss/5.0.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: emboss
+  version: "5.0.0"
+
+build:
+  number: 0
+  skip: False
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
```

File: recipes/emboss/5.0.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+source:
+  fn: emboss-5.0.0-Linux-x86_64.tgz     # [linux]
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
```

File: recipes/emboss/5.0.0/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  url: http://depot.galaxyproject.org/package/linux/x86_64/emboss/emboss-5.0.0-Linux-x86_64.tgz     # [linux]
+  sha256: 44fc68cf43f86a039b45537f3d43aa6bcf9f36bbe707ddd42519b6055c17b95a     # [linux]
+  fn: emboss-5.0.0-Darwin-x86_64.tgz    # [osx]
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
```

File: recipes/emboss/5.0.0/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  url: http://depot.galaxyproject.org/package/darwin/x86_64/emboss/emboss-5.0.0-Darwin-x86_64.tgz   # [osx]
+  sha256: 12d85568440af4d5a9f8c4f0fef439c70a3ac4c380b3d3116b827b8e839e385f # [osx]
+
+requirements:
+  build:
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
  host:
```

File: recipes/emboss/5.0.0/meta.yaml [hunk 5]
```diff
@@ -4,6 +4,7 @@ package:
 
 build:
   number: 0
+  noarch: generic
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]: From the linter messages: "The recipe uses per platform sources and cannot be noarch". We need to somehow skip the check for noarch, but this is wrong I think.

**Comment by nsoranzo** [comment]
Need to remove `recipes/emboss` from build-fail-blacklist

File: recipes/emboss/5.0.0/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  home: http://emboss.open-bio.org/
+  license: GPL
+  summary: The European Molecular Biology Open Software Suite 
+
+test:
+  commands:
+    -  needle --help
+    -  transeq --help
+    -  sixpack --help
+    -  seqret --help
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
    -  seqret --help

extra:
  skip-lints:
    - should_be_noarch_generic
```

File: recipes/emboss/5.0.0/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: emboss
+  version: "5.0.0"
+
+build:
+  number: 0
+  noarch: generic
```
Comment (nsoranzo) [inline recipes/emboss/5.0.0/meta.yaml]:
```suggestion
```

**Comment by mvdbeek** [comment]
No idea what's going on with the OSX build. It does fail with the same error trying to build locally.

**Comment by mvdbeek** [comment]
Alright it's green, anyone ready to push the button ?

**Comment by nsoranzo** [comment]
@BiocondaBot please merge

**Pull request merged**